### PR TITLE
Partner: Scuba Analytics

### PR DESCRIPTION
**Pull request by internetisaiah**
# Pull Request/Issue Resolution

#### Description of Change:

Scuba Analytics Partner Documentation from @scuba-tptns. I moved this over from #6477 because there was an issue with the CLA bot.

#### Is this change associated with a Braze feature/product release?

- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by internetisaiah** [comment]
Hey @scuba-tptns, I'm going to make some slight changes to your screenshots. Do mind sending me the raw `.png` files without any edits/markup to them (like the red boxes/arrows)?

Thanks in advance!

File: _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+| `PERIOD_START`          | The start period as a [BQL-compatible][6] date.                                                                                                                                              |
+| `PERIOD_END`            | The end period as a [BQL-compatible][6] date.                                                                                                                                                |
+| `RECORD_LIMIT`          | **Optional**: The maximum number of records to return. By default, Scuba returns a maximum of 100 records. You can change this by assigning any non-negative number to `scuba_record_limit`. |
+{: .reset-td-br-1 .reset-td-br-2}
+
+### Default behavior
+
+By default, `update_existing_only` is set to `false` which will update your existing records in Braze, as well as create new records for those that don't exist. To prevent Scuba from creating new records, set `update_existing_only` to `true`.
+
+### Rate limit
```
Comment (internetisaiah) [inline _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md:70]: @lydia-xie , when you get the chance, can you verify that this rate limit is correct? Also I see that we have a bunch of rate limits in the `_includes` directory. Is there one I can pull from there so we have unified language?

Thanks in advance!

File: _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+article_title: Scuba Analytics
+description: "This Scuba and Braze technical reference describes how to activate Scuba's real-time data insight using Braze Segments."
+alias: /partners/scuba/
+page_type: partner
+search_tag: Partner
+---
+
+# Scuba Analytics
+
+>[Scuba Analytics][1] is a full-stack, machine-learning-powered data collaboration platform designed for high-velocity time-series data. It redefines how organizations handle first-party and high-velocity time-series data, offering insights into customers and real-time competition tracking.
```
Comment (internetisaiah) [inline _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md:12]: Hey @scuba-tptns, I cleaned up the intro and cut down some of the content here. Can you verify that my changes did not alter the intended meaning of anything here?

Additionally, is it supposed to be "time-series data" or "time-serialized data"? I've not heard the term "time-series" before, so I wasn't sure.

Also, if there's anything I cut that you think should be added, let me know and I'd be happy to re-add some of that content in.

File: _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Prerequisites
+
+To use Scuba Analytics with Braze, you'll need the following:
+
+- A [Braze REST API key]() with the `users.track` permission.
+- A [Scuba API token]() with the `TODO` permission.
```
Comment (internetisaiah) [inline _docs/_partners/data_and_infrastructure_agility/analytics/scuba.md:19]: @scuba-tptns, also, is there a required permission level for the Scuba API token? Or is a token with basic permissions fine?

**Comment by internetisaiah** [comment]
Hey @josh-mccrowell-braze , this one's ready for review! Let me know if there's any clarification you need from me. Otherwise, we can merge.

cc @scuba-tptns

**Review by josh-mccrowell-braze — APPROVED** [review]
Looks good.  I massaged a little bit of "marketingy" messaging, but it was a minor change. Appreciate the technical thoroughness.

**Pull request merged**